keyboards/hasupad/keymaps/default/keymap.c

```diff
@@ -0,0 +1,122 @@
+
```

Comment:
empty lines should be in groups of 1 and the file should start with a line, not an empty line